sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/QueuePropertiesExtensions.cs

```diff
@@ -107,65 +112,73 @@ private static async Task<QueueProperties> ParseFromEntryElementAsync(XElement x
                     case "RequiresSession":
                         properties.RequiresSession = bool.Parse(element.Value);
                         break;
+                    case "DefaultMessageTimeToLive":
+                        properties.DefaultMessageTimeToLive = XmlConvert.ToTimeSpan(element.Value);
+                        break;
                     case "DeadLetteringOnMessageExpiration":
                         properties.DeadLetteringOnMessageExpiration = bool.Parse(element.Value);
                         break;
                     case "DuplicateDetectionHistoryTimeWindow":
                         properties.DuplicateDetectionHistoryTimeWindow = XmlConvert.ToTimeSpan(element.Value);
                         break;
-                    case "LockDuration":
-                        properties.LockDuration = XmlConvert.ToTimeSpan(element.Value);
-                        break;
-                    case "DefaultMessageTimeToLive":
-                        properties.DefaultMessageTimeToLive = XmlConvert.ToTimeSpan(element.Value);
-                        break;
                     case "MaxDeliveryCount":
                         properties.MaxDeliveryCount = int.Parse(element.Value, CultureInfo.InvariantCulture);
                         break;
                     case "EnableBatchedOperations":
                         properties.EnableBatchedOperations = bool.Parse(element.Value);
                         break;
-                    case "Status":
-                        properties.Status = element.Value;
-                        break;
-                    case "AutoDeleteOnIdle":
-                        properties.AutoDeleteOnIdle = XmlConvert.ToTimeSpan(element.Value);
+                    case "IsAnonymousAccessible":
+                        properties.IsAnonymousAccessible = Boolean.Parse(element.Value);
                         break;
-                    case "EnablePartitioning":
-                        properties.EnablePartitioning = bool.Parse(element.Value);
+                    case "AuthorizationRules":
```

Comment:
Not worth taking on this close to code complete, but may be worth a follow-up at some point to pull the magic strings out into a set of constants.